cfgov/cfgov/settings/base.py

```diff
@@ -197,7 +198,8 @@
 # Used to include directories not traditionally found,
 # app-specific 'static' directories.
 STATICFILES_DIRS = [
-    PROJECT_ROOT.child('static_built')
+    PROJECT_ROOT.child('static_built'),
+    PROJECT_ROOT.child('templates', 'wagtailadmin')
```

Comment:
> Why add the wagtail admin templates to the static dirs?

We are adding Javascript and CSS to extend the Wagtail admin. Wasn't sure if there was a more appropriate place to place the assets.